core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -865,17 +907,24 @@ public void operationComplete(ChannelFuture f) throws Exception {
     }
 
     private static String bossThreadName(ServerPort port) {
-        final InetSocketAddress localAddr = port.localAddress();
-        final String localHostName =
-                localAddr.getAddress().isAnyLocalAddress() ? "*" : localAddr.getHostString();
-
         // e.g. 'armeria-boss-http-*:8080'
         //      'armeria-boss-http-127.0.0.1:8443'
         //      'armeria-boss-proxy+http+https-127.0.0.1:8443'
+        //      'armeria-boss-http-unix:/var/run/server.sock'
         final String protocolNames = port.protocols().stream()
                                          .map(SessionProtocol::uriText)
                                          .collect(Collectors.joining("+"));
-        return "armeria-boss-" + protocolNames + '-' + localHostName + ':' + localAddr.getPort();
+        final InetSocketAddress localAddr = port.localAddress();
+        final String localAddrText;
+        if (!port.isDomainSocket()) {
+            localAddrText =
+                    (localAddr.getAddress().isAnyLocalAddress() ? "*" : localAddr.getHostString()) +
+                    ':' + localAddr.getPort();
+        } else {
+            localAddrText = localAddr.toString();
```

Comment:
Should we prepend `unix:` to `localAddrText` to abide by the above pattern `armeria-boss-http-unix:/var/run/server.sock`? `localAddr.toString()` does not start with `unix:`.
<img width="863" alt="image" src="https://github.com/line/armeria/assets/1866157/306ad0db-3e5c-41ff-9753-7afcd15c390b">

Reply:
Agreed. Let me add the `unix:` prefix. :bow: